pyproject.toml

```diff
@@ -63,7 +63,7 @@ deltalake = { version = ">=0.9.0,<1", optional = true }
 duckdb = { version = ">=0.8.1,<1", optional = true }
 duckdb-engine = { version = ">=0.1.8,<1", optional = true }
 fsspec = { version = ">=2022.1.0", optional = true }
-geoalchemy2 = { version = ">=0.6.3,<1,!=0.13.0,!=0.14.0,!=0.14.1", optional = true }
```

Comment:
These geoalchemy2 versions work with the postgres geo functionality, but there's a strange issue about a missing `SRS_ID` column that only shows up with:

1. The MySQL backend
2. `SQLAlchemy < 2`
3. `Geoalchemy2 >= 0.14`

So I elected to remove the constraints here to let people use a larger set of versions.